addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1278,16 +1275,13 @@ const Wysiwyg = Widget.extend({
         // selection when the modal is closed.
         const restoreSelection = preserveCursor(this.odooEditor.document);
 
-        const $editable = $(OdooEditorLib.closestElement(range.startContainer, '.o_editable') || this.odooEditor.editable);
-        const model = $editable.data('oe-model');
-        const field = $editable.data('oe-field');
-        const type = $editable.data('oe-type');
+        const editable = OdooEditorLib.closestElement(range.startContainer, '.o_editable') || this.odooEditor.editable;
```

Comment:
@dmo-odoo 
No, because it was done elsewhere.
saas-16.4+: https://github.com/odoo/odoo/pull/135880
Up to saas-16.3: https://github.com/odoo/odoo/commit/a46a8f45b65ca9ecfbb12ffb7980902085666e1a

Reply:
The `params.node` part of the fix, yes, but not the closest with jQuery data.

Reply:
This is how I understand the whole thing:

The point of `closest`/`closestElement` was to find the `.o_editable` element from which we can extract data referring to a record (res_model, res_id, etc).
Such selector (`.o_editable`) is:
  - good for website, as the data attributes are there (they come from QWeb rendering as regular html data attributes).
  - bad for mass_mailing, as such data attributes are not there. 
 
 So the solution in the PR you refer to was to include those data attributes in the selector so that in mass_mailing `closest` would miss the `.o_editable` element, and it would fall back to the `odooEditor.editable` element instead. Such element contains the res_model and res_id in its jQuery dataset, so that works for mass_mailing or any html field.

The whole problem resides in `openMediaDialog` having to get the correct record info for both HtmlField and website, even though such data is available through different forms. A similar problem exists for `saveModifiedImages`.

This is what **this** PR proposed instead: an overridable `_getRecordInfo` method to get the record data, in which it is fetched from `options.recordInfo` (this works when the wysiwyg is instantiated by the HtmlField component). Website, having different needs (multiple editables) and having the record info stored as regular data attributes** that come from QWeb, overrides such method. Note that the `editable` element resulting of the `closestElement` call is not important for the default (HtmlField) case. We don't need to fetch oe-model and oe-id from the editable's jQuery dataset, as they were set using the info from  `options.recordInfo` in the first place (see Wysiwyg.startEditor). We can get it directly from the source instead. The `editable` is only relevant for the website override.

I am probably missing something here, so I ask: what problem are we trying to solve by incorporating the fix you refer to? Has it not been solved by this PR (and the other one that fixes the "Replace media" behavior https://github.com/odoo/odoo/pull/135880)?

Lastly, I do not understand the role of `data(oe-media-domain)` as an alternative selector it the PR you refer to. But `oe-media-domain` is nowhere to be found in the codebase, and the `domain` prop does not seem to be used by the `MediaDialog` component. And It was decided to remove it here: https://discord.com/channels/678381219515465750/687338152372994236/1158690890366332948

Anyway, increasing the selector's specificity (with `:data(...)`) would do no harm (other than increasing dependency on jQuery). If it is important and you want *me* to do it, I just need to know why, so I can justify it in the commit message.

**For that matter, using jQuery to access such data was not necessary, and my original commit dropped its use, but you asked me to bring it back (see https://github.com/odoo/odoo/pull/137546#discussion_r1351417140 and https://github.com/odoo/odoo/compare/4b0f6a04ee3a92b5300177b7c2192f422f49f641..c4c869e0d3efee89c31c9c05d786013d4b77fdd6).